fs/rails-base-graphql-api
# Add Event attribute to Activities

### Summary

This PR adds `event` enum to Activity model. It will allow us to filter activities by event type.

### How it works

![Screen Shot 2020-06-15 at 11 01 02](https://user-images.githubusercontent.com/58729845/84632642-e0ce6800-aef7-11ea-9cf5-006d82edc932.png)

### Test plan

List of steps to manually test introduced functionality:

* Open GraphiQL App
* Make request using schema:
```
query Activities {
  activities {
    id
    title
    body
    event
    createdAt
  }
}
```
* Check that response have items with `"event": "user_registered"`

### Review notes

While reviewing pull-request (especially when it's your pull-request),
please make sure that:

- you understand what problem is solved by PR and how is it solved
- new tests are in place, no redundant tests
- DB schema changes reflect new migrations
- newly introduced DB fields have indexes and constraints
- there are no missed files (migrations, view templates)
- required ENV variables added and described in `.env.example` and added to Heroku
- associated Heroku review app works correctly with introduced changes

### Deploy notes

This PR contains changed DB migrations. After deploy we need to re-create database:
```
heroku run rails db:drop db:setup
```

### References
* Resolves #53
